meson.build

```diff
@@ -162,15 +163,70 @@ if not glib_dep.found()
   glib_internal = true
   meson_cmd = find_program('meson')
   ninja = find_program('ninja')
+
   glib_internal_download_t = custom_target('glib-internal-download',
     command : [ meson_cmd, 'subprojects', 'download', 'glib', '--sourcedir', meson.current_source_dir() ],
     console : true,
     output : ['glib-internal-download'],
   )
+
+  glib_internal_dependencies = [
+    dependency('threads'),
+  ]
+  glib_internal_configure_args = []
+
+  glib_internal_local_lib = false
+  if cc.has_function('iconv_open')
+    # pass
+  else
```

Comment:
Wait does this language not support something like `if not expr`?

Reply:
actually.. it does. fixed